metadata.py

```
@@ -642,17 +644,16 @@ def on_deleted(self, event: FileSystemEventHandler) -> None:
         logging.debug(f"Detected a file deletion: {event.src_path}")
 
     def is_file_ready(
-        self, 
-        filepath: str, 
-        stable_time: int = 5, 
-        check_interval: int = 1
-        ) -> bool:
+        self, filepath: str, stable_time: int = 5, check_interval: int = 1
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)